### PR TITLE
[cookbook] fix broken cove notebook

### PR DESCRIPTION
**Pull request by tanya-rai**
[cookbook] fix broken cove notebook

- fixed broken cove notebook
- updated to be a template not just demo
- updated readme

### CODE REVIEW

File: cookbooks/Chain-of-Verification/cove_template_config.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      "name": "baseline_response_gen",
+      "input": "{{baseline_prompt}}",
+      "metadata": {
+        "model": {
+          "name": "gpt-4",
+          "settings": {
+            "system_prompt": ""
+          }
+        },
+        "parameters": {},
```
Comment (saqadri) [inline cookbooks/Chain-of-Verification/cove_template_config.json:30]: Needed?

File: cookbooks/Chain-of-Verification/cove_template_config.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  },
+  "prompts": [
+    {
+      "name": "baseline_response_gen",
+      "input": "{{baseline_prompt}}",
+      "metadata": {
+        "model": {
+          "name": "gpt-4",
+          "settings": {
+            "system_prompt": ""
```
Comment (saqadri) [inline cookbooks/Chain-of-Verification/cove_template_config.json:27]: Needed?

**Review by saqadri — APPROVED** [review]
Approving to unblock, some comments

**Comment by tanya-rai** [comment]
Made the config with workbook - didn't add manually. Will remove though to make it cleaner

**Comment by tanya-rai** [comment]
Tried removing those and lines it breaks - will merge the original and try to debug separately. 

Screenshot 2023-11-24 at 2 17 07 PM

**Comment by saqadri** [comment]
Yeah would be good to file an issue -- cc @Ankush-lastmile. There is some weirdness with parameters and system prompt being set to empty in this config

**Comment by jonathanlastmileai** [comment]
> weirdness with parameters and system prompt being set to empty in this config

I volunteered to take the lead on py bug fixes, cleanup, testing. https://github.com/lastmile-ai/aiconfig/issues/327

**Pull request merged**